pkg/forwarder/domain_forwarder.go

```diff
@@ -21,19 +21,22 @@ var (
 	chanBufferSize = 100
 	flushInterval  = 5 * time.Second
 
-	transactionsRetried  = expvar.Int{}
-	transactionsDropped  = expvar.Int{}
-	transactionsRequeued = expvar.Int{}
-
-	tlmTxRetried = telemetry.NewCounter("transactions", "retries",
-		[]string{"domain"}, "Transaction retry count")
-	tlmTxDropped = telemetry.NewCounter("transactions", "dropped",
-		[]string{"domain"}, "Transaction drop count")
-	tlmTxRequeud = telemetry.NewCounter("transactions", "requeud",
-		[]string{"domain"}, "Transaction requeue count")
+	transactionsRetried  = expvar.Map{}
+	transactionsDropped  = expvar.Map{}
+	transactionsRequeued = expvar.Map{}
+
+	tlmTxRetried = telemetry.NewCounter("forwarder_transactions", "retries",
```

Comment:
Is it OK to remove existing telemetry counters?

Reply:
What do you mean? This is not removed.

Reply:
You renamed `"transactions"` to `"forwarder_transactions"` (Before you had `tlmTxRetried = telemetry.NewCounter("transactions", "retries", []string{"domain"}, "Transaction retry count")`.
It is OK to rename a exp var? Can it break something?

Reply:
Not sure. I will have to update dashboards I think.